### ISSUE TITLE
Paper : Distilling Knowledge from Noisy Teachers

### ISSUE THREAD

**Issue by NeelayS** [SforAiDl/KD_Lib#13]
Link - https://arxiv.org/pdf/1610.09650.pdf

Simulates effect of using multiple teachers by adding gaussian noise to teacher logits.

**Comment by Het-Shah** [comment]
Hey, Great paper!
